src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -27,6 +30,8 @@
 )
 from neuroconv.utils import NWBMetaDataEncoder
 
+interfaces_for_testing_probe = [IntanRecordingInterface]
```

Comment:
Since this PR is to add a general method to all `BaseRecordingExtractor`, we really should ensure to the absolute best of our ability that it works with all of the child classes with example data we claim to support, not just the one that started the discussion

I'd really love to see this applied to all the interfaces that rely on this mixin - happy to help with debugging the various cases that can occur throughout this process!